v1/traits/kv.cue

```diff
@@ -9,7 +9,7 @@ import (
 	$metadata: traits: Redis: null
 	redis: {
 		name:    string | *$metadata.id
-		version: string | *"7.0"
+		version: string | int | *"7.0"
```

Comment:
I think this should stay a string, to support semantic versioning

Reply:
if you need an int you can use `strconv.Atoi(string)` to convert the string to int
you can find the list of available cue helper functions here https://pkg.go.dev/cuelang.org/go/pkg